cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRollingTest.java

```diff
@@ -344,8 +444,11 @@ public void shouldRestartMultipleBrokersWithTopicWithNoReason() throws Execution
                 3,
                 5);
 
-        // then
```

Comment:
No need to remove this `// then`, since all the tests are using these comments to delineate the common structure.